atlas_densities/densities/inhibitory_neuron_density.py

```diff
@@ -138,7 +140,7 @@ def compute_inhibitory_neuron_density(  # pylint: disable=too-many-arguments
                 "Either inhibitory_proportion or inhibitory_data should be provided"
                 ". Both are None."
             )
-        group_ids = get_group_ids(region_map)
+        group_ids = get_group_ids(region_map, root_region_name=root_region_name)
```

Comment:
we should raise an exception or assert if we don't find a root no ?

Reply:
yup, completely agree